apps/server/src/modules/authentication/loggable/user-account-deactivated-exception.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpStatus } from '@nestjs/common';
+import { BusinessError } from '@shared/common';
+import { ErrorLogMessage, Loggable } from '@src/core/logger';
+
+export class UserAccountDeactivatedException extends BusinessError implements Loggable {
```

Comment:
Please rename it to `UserAccountDeactivatedLoggableException` (just add the `Loggable` in between) to keep the naming convention for the loggable exceptions.